playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml

```diff
@@ -21,15 +21,23 @@
   gather_facts: false
   connection: local
   tasks:
+
+    - name: Remove ssh key entry
+      shell: "ssh-keygen -R {{ hostvars[item].nodepool.private_ipv4 }}"
+      args:
+        executable: /bin/bash
+      when: item in ["iosxr-7.0.2"]
+      with_inventory_hostnames: appliance
+
     # NOTE(pabelanger): This is a huge hack to deal with the fact there is no
     # default gateway seutp.
     - name: Ensure we can SSH into the appliance
       expect:
         command: "ssh admin@{{ hostvars[item].nodepool.private_ipv4 }}"
         responses:
           'Are you sure you want to continue connecting': "yes"
-          'password:': admin
-          'RP/0/0/CPU0:ios#': exit
+          '(?i)password:': admin123
```

Comment:
we should revert this back to admin, so we don't have to have 2 playbooks for iosxr. This likely means, re-rolling the iosxr images to default to admin.

Reply:
@pabelanger iosxr7.0.2 has restrictions on minimum password length which is 6. So we will not be able to use admin. I shall add a check here, for 7.0.2

Reply:
k, if that is the case we can move to inventory vars for, which are loaded depending on which image.